src/name_gen.rs

```diff
@@ -0,0 +1,15 @@
+pub fn generate_simple_name(epic_title: String) -> String {
```

Comment:
Consider taking an `&str` for more flexibility, since then we can also accept a reference to a `String` (by deref coercions).